src/components/views/elements/ImageView.tsx

```diff
@@ -159,8 +161,8 @@ export default class ImageView extends React.Component<IProps, IState> {
             });
             return;
         }
-        if (newZoom >= this.state.maxZoom) {
-            this.setState({zoom: this.state.maxZoom});
+        if (newZoom >= 2 * this.state.maxZoom) {
+            this.setState({zoom: 2 * this.state.maxZoom});
```

Comment:
Could you perhaps export this into a const and explain why are we multiplying by 2?

Reply:
Sure, I'll add a comment.